### PR TITLE
Add options for cleaner excelfile

### PR DESCRIPTION
**Pull request by lsoest**
Hi guerinoni,

I needed an excel file with just the strings and translations:

1. added two command line switches to disable locations/version info. (not sure if this is the way to go)
2. enabled tests on windows machine
3. added debug output for some tests

### CODE REVIEW

File: .github/workflows/w10.yaml [hunk 1]
```diff
@@ -25,9 +25,19 @@ jobs:
...
+          ctest --output-on-failure
+
+      - name: Build CLI
+        shell: pwsh
+        run: |
+          mkdir build-cli
+          cd build-cli
+          cmake -DBUILD_CLI:BOOL=ON ..
+          cmake --build . --parallel
+          ctest --output-on-failure
```
Comment (guerinoni) [inline .github/workflows/w10.yaml:43]: Many thanks, I don't have w10 :)

**Review by guerinoni — COMMENTED** [review]
Seems all good! Can you add some documentation in Readme about this new parameters, how use it from CLI and so on? :)

**Comment by lsoest** [comment]
Added some documentation for the flags and how to build & install for windows.
Added to cmake to also install the required dlls to the correct location.

**Review by guerinoni — APPROVED** [review]
LGTM! :)

**Pull request merged**